go/src/koding/klient/app/klient.go

```diff
@@ -777,7 +777,7 @@ func (k *Klient) Run() {
 
 	// TODO(rjeczalik): Enable after TMS-848.
 	if k.autoupdateEnabled() {
-		go k.updater.Run()
+		//go k.updater.Run()
```

Comment:
@rjeczalik this is wip I made this change temporary in order to use it on my remote machine for testing. Will remove this after all the code is done.